src/bindings/python/src/compatibility/ngraph/__init__.py

```diff
@@ -193,6 +193,13 @@
 from ngraph.opset11 import unsqueeze
 from ngraph.opset11 import variadic_split
 
+import warnings
+
+warnings.warn(
+    message="OpenVINO Python API 1.0 is deprecated and will be removed in 2024.0 release.",
```

Comment:
Let's add a link to migration guide as it done for C++: https://github.com/openvinotoolkit/openvino/blob/master/src/inference/include/ie/ie_api.h#L56

Reply:
added